sos/report/plugins/networkmanager.py

```diff
@@ -110,16 +117,19 @@ def test_nm_status(version=1):
             "nmcli dev show .*": "nmcli_dev_show_sos"
         })
 
-    def postproc(self):
-        for _, _, files in os.walk(
-                "/etc/NetworkManager/system-connections"):
+    def postproc_system_connections(self, sc_path):
+        for _, _, files in os.walk(sc_path):
```

Comment:
Why not use `do_path_regex_sub()` ?
https://github.com/sosreport/sos/blob/279ea13ce25ef5b60e9b3a120d6a2add8e962f16/sos/report/plugins/kubernetes.py#L291-L292

Reply:
good catch, I think that would make sense, that would cover all files in the directory